test/Organizer/ProcessManager/GeoCoordinatesProcessManagerTest.php

```diff
@@ -12,6 +12,7 @@
 use CultuurNet\UDB3\Event\GeoCoordinatesProcessManager as GeoCoordinatesProcessManagerAlias;
 use CultuurNet\UDB3\Organizer\Commands\UpdateGeoCoordinatesFromAddress;
 use CultuurNet\UDB3\Organizer\Events\AddressUpdated;
+use CultuurNet\UDB3\Organizer\ProcessManager\GeoCoordinatesProcessManager;
```

Comment:
Is this `use` statement necessary? Both the test and the process manager use the same namespace.